clients/src/main/java/org/apache/kafka/common/requests/RequestContext.java

```diff
@@ -37,6 +37,7 @@ public class RequestContext implements AuthorizableRequestContext {
     public final RequestHeader header;
     public final String connectionId;
     public final InetAddress clientAddress;
+    public final Optional<Integer> clientPort;
```

Comment:
I have kept the client port as Optional because the data should not be required for forwarded requests in context of KIP-714.